docs/pages/platform/upgrading/1.2.mdx

```diff
@@ -5,257 +5,385 @@ meta:
+single-room token based setup.
+
+<Steps>
+
+  <Step>
+    <StepTitle>Create a client</StepTitle>
+    <StepContent>
+    Create a Node.js client that allows you to interact with our REST API.
+      ```tsx
+      import { Liveblocks } from "@liveblocks/node";
+      
+      const liveblocks = new Liveblocks({
+        secret: "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+      });
+      ```
+    </StepContent>
+  </Step>
+  
+  <Step>
+    <StepTitle>Start an auth session inside your endpoint</StepTitle>
+    <StepContent>
+    Every session should have a unique user ID, which is typically the ID of the user in your database.
+      ```tsx highlight="10-13"
+      import { Liveblocks } from "@liveblocks/node";
+      
+      const liveblocks = new Liveblocks({
+        secret: "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+      });
+      
+      export function POST() {
+        const user = { id: "olivier@example.com", info: { name: "Olivier" }};
+  
+        const session = liveblocks.prepareSession(
+          user.id,
+          { userInfo: user.info } // Optional
+        );
+      }
+      ```
+    </StepContent>
+  </Step>
+  
+  <Step>
+    <StepTitle>Give the user access to the room</StepTitle>
+    <StepContent>
+    Give if the current user access to the room, with either `session.FULL_ACCESS`, or `session.READ_ACCESS`.
+      ```tsx highlight="15"
+      import { Liveblocks } from "@liveblocks/node";
+      
+      const liveblocks = new Liveblocks({
+        secret: "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+      });
+      
+      export function POST() {
+        const user = { id: "olivier@example.com", info: { name: "Olivier" }};
+  
+        const session = liveblocks.prepareSession(
+          user.id,
+          { userInfo: user.info } // Optional
+        );
+
+        session.allow("my-room-name", session.FULL_ACCESS);
+      }
+      ```
+    </StepContent>
+
+  </Step>
+  
+  <Step>
+    <StepTitle>Authorize the user and return the result</StepTitle>
+    <StepContent>
+    Give if the current user access to the room, with either `session.FULL_ACCESS`, or `session.READ_ACCESS`.
+      ```tsx highlight="17,18"
+      import { Liveblocks } from "@liveblocks/node";
+      
+      const liveblocks = new Liveblocks({
+        secret: "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+      });
+      
+      export function POST() {
+        const user = { id: "olivier@example.com", info: { name: "Olivier" }};
+  
+        const session = liveblocks.prepareSession(
+          user.id,
+          { userInfo: user.info } // Optional
+        );
+
+        session.allow("my-room-name", session.FULL_ACCESS);
+
+        const { status, body } = await session.authorize();
+        return body;
+      }
+      ```
+    </StepContent>
+
+  </Step>
+  
+  <Step>
+    <StepTitle>Point to the new endpoint</StepTitle>
+    <StepContent>
+    Update your `liveblocks.config.ts` file to connect to the new endpoint.
+    ```ts file="liveblocks.config.ts" highlight="2"
```

Comment:
Is it clear enough that this part should happen in the front-end? Maybe add a comment in the code block to extra emphasize that?

Reply:
Good idea. I've written "In the front end of your app, update your..." and also shown the `@liveblocks/client` import in the code.